index.html

```diff
@@ -173,6 +173,25 @@ <h1>Multi-Post <br>Stories</h1>
                 </li>
             </ul>
         </section>
+
+        <section class="Contact-form">
+            <div>
+                <div class="contact-title">Contact me</div>
+                <div class="contact-content">
+                    If you have an application you <br> are interested in 
+                    developing, a <br> feature that you 
+                    need built or a <br> project that needs coding. I’d <br> love to help with it 
+                </div>
+                <form action="https://formspree.io/f/{form_id}" method="POST">
+                    <ul>
+                        <li><input type="text" name="for-name" id="name" placeholder="Put your name" maxlength="30" required></li>
+                        <li><input type="email" name="for-mail" id="mail" placeholder="put your adress email" required></li>
+                        <li><textarea name="for-text" id="textarea" maxlength="500" cols="30" rows="10" placeholder="Write your message here" required></textarea></li>
+                        <li><button>Get it Touch</button></li>
+                    </ul>            
+                </form>
+            </div>
+        </section>
```

Comment:
- [ ] Great job implementing the contact form but one of the project requirements is that the contact form is in the `footer`. Kindly fix that.

![image](https://user-images.githubusercontent.com/59416136/155171707-b6b3f8d1-bc0a-4a52-9adf-46c29e3bd050.png)